### DISCUSSION TITLE
Automatic translation across components of different projects?

### DISCUSSION THREAD

**Discussion by JulianGro** [WeblateOrg/weblate#7200] [Q&A] [answered]
Hello,

we have two projects that forked. Can we share translations without needing to be in the same project inside Weblate?

**Comment by nijel** [comment, marked as answer]
Yes, you can setup automatic translation add-on for that. Only person being admin in both projects will be able to do that.

**Reply by JulianGro** [reply]
Ah okay. I was testing it on a random other project so now it makes sense.

**Reply by m-aciek** [reply]
@nijel Could you confirm that it can be done by:
* Setting a source project to use Weblate shared translation memory,
* Configuring Automatic translation add-on in components to use "Machine translations" and selecting "Weblate Translation Memory" as engine?

**Reply by nijel** [reply]
Choose "Other translation components" as source and select the ones you want, see https://docs.weblate.org/en/latest/user/translating.html#auto-translation

**Reply by m-aciek** [reply]
How would I select components from the other project?

**Reply by JulianGro** [reply]
> Yes, you can setup automatic translation add-on for that. Only person being admin in both projects will be able to do that.

I am admin in both projects, but cannot select the other component. There is only a text box which gets me a "component not found" when I put a component in with the `project/component` format.

**Reply by nijel** [reply]
The project of the target component also needs to have [Contribute to shared translation memory](https://docs.weblate.org/en/latest/admin/projects.html#contribute-to-shared-translation-memory) enabled.

**Reply by Corentvn** [reply]
Hello @nijel, 

> Choose "Other translation components" as source and select the ones you want


On the UI we read 

> Enter slug of a component to use as source, keep blank to use all components in the current project.


What if we want it to be from all components of **all** projects? Or for all components of **a subset of all** projects?

Thank you

**Reply by nijel** [reply]
The translation memory is then the way to go.
